index.html

```diff
@@ -107,40 +107,58 @@ <h3>Multi-Post <br> Stories</h3>
+        <span>LET'S CONNECT</span>
+        <ul class="media-links">
+            <li><a href="#"><i class="fa fa-twitter"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-linkedin-in"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-m"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li> 
+        </ul>
+        <button class="resume-mobile-btn" type="submit">Get my resume</button>
+        <ul class="lang-area">
+            <ul class="stacknav">
+                <li><strong>Languages</strong></li>
+                <li><i class="fa-solid fa-angle-down"></i></li>
             </ul>
-            <button class="resume-mobile-btn" type="submit">Get my resume</button>
-            <ul class="lang-area">
-                <ul class="stacknav">
-                    <li><strong>Languages</strong></li>
-                    <li><i class="fa-solid fa-angle-down"></i></li>
-                </ul>
-                <ul class="stacks">
-                    <li><button class="html" type="submit"><a href="#">HTML</a></button></li>
-                    <li><button class="css" type="submit"><a href="#">CSS</a></button></li>
-                    <li><button class="js" type="submit"><a href="#">JavaScript</a></button></li>
+            <ul class="stacks">
+                <li><button class="html" type="submit"><a href="#">HTML</a></button></li>
+                <li><button class="css" type="submit"><a href="#">CSS</a></button></li>
+                <li><button class="js" type="submit"><a href="#">JavaScript</a></button></li>
+            </ul>
+            <ul>
+                <ul class="frameworks">
+                    <li><strong>Frameworks</strong></li>
+                    <li><i class="fa-solid fa-angle-right"></i></li>
                 </ul>
-                <ul>
-                    <ul class="frameworks">
-                        <li><strong>Frameworks</strong></li>
-                        <li><i class="fa-solid fa-angle-right"></i></li>
-                    </ul>
-                    <ul class="skills">
-                        <li><strong>Skills</strong></li>
-                        <li><i class="fa-solid fa-angle-right"></i></li>
-                    </ul>
+                <ul class="skills">
+                    <li><strong>Skills</strong></li>
+                    <li><i class="fa-solid fa-angle-right"></i></li>
                 </ul>
             </ul>
-        </section>
+        </ul>
+    </section>
+        <!-- contact me  -->
+    <section class="contact">
+        <div class="contact-bg-img">
+            <h6>Contact me</h6>
+            <p class="contact-p">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+            I’d love to help with it!</p>
+            <form action="">
```

Comment:
- [ ] Nice work implementing the form, it look really good :clap: However it states in the project requirement that data should be collected by using the [Formspree service](https://formspree.io/html). Therefore I suggest you:
  - add the `action` attribute to your form tag and its value should be similar to `https://formspree.io/f/{form_id}` where `{form_id}`  is `id` giving you when create an account on [formspree service](https://formspree.io/html).
  - add the `method` attribute to your `form` tag and its value  equals `POST`